### PR TITLE
Cart Screen

### PR DESCRIPTION
**Pull request by m26unkwn**
1. Poducts
2. Place order Card

### CODE REVIEW

File: screens/mycart.html [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+        </div>
+        <section class="pd-container-main flex flex-wrap flex-gap jc-center">
+          <div class="pd-wrapper flex jc-center flex-col flex-gap flex-wrap">
+            <div class="pd-card-container">
+              <div class="card-img-wrapper">
+                <a href="#">
+                  <img
+                    src="../assets/Tees-Transparent.png"
+                    class="pd-img card-img vertical-card-img"
+                    alt="img"
```
Comment (ashishkatiyar2404) [inline screens/mycart.html:164]: alt values should be meaning full.

File: screens/mycart.html [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+                      stroke-linejoin="round"
+                    />
+                  </svg>
+                </button>
+              </div>
+              <div class="pd-content">
+                <h1 class="pd-heading card-heading">Keychron K2 V-2</h1>
+                <p class="pd-desc">mechanical keyboard</p>
+                <div class="pd-price">
+                  <p class="crnt-price">$200</p>
```
Comment (ashishkatiyar2404) [inline screens/mycart.html:204]:
```suggestion
                  <p class="current-price">$200</p>
```

File: screens/mycart.html [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+                </div>
+              </div>
+            </div>
+            <div class="pd-card-container">
+              <div class="card-img-wrapper">
+                <a href="#">
+                  <img
+                    src="../assets/Tees-Transparent.png"
+                    class="pd-img card-img vertical-card-img"
+                    alt="img"
```
Comment (ashishkatiyar2404) [inline screens/mycart.html:282]: alt value should be meaningful

File: styles/cart.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.quantity-view {
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+
+  padding: 10px;
+
+  width: calc(var(--spacing-md) * 2);
+  height: calc(var(--spacing-md) * 2);
```
Comment (ashishkatiyar2404) [inline styles/cart.css:16]: appreciation:-  good to see this you gave a space which is related to border thn  length.

**Review by ashishkatiyar2404 — APPROVED** [review]
suggested some changes pls make changes before merging.
rest looks good to merge💯

File: styles/cart.css (hunk 4 above)
Comment (m26unkwn) [reply styles/cart.css:16]: Thanks @ashishkatiyar2404

**Pull request merged**